Lib/test/test_tools/test_bundle_ensurepip_wheels.py

```diff
@@ -0,0 +1,100 @@
+import sys
+import tempfile
+import unittest
+import unittest.mock
+import urllib.request
+from hashlib import sha256
+from io import BytesIO
+from pathlib import Path
+from random import randbytes
+from test import support, test_tools
+from test.support import captured_stderr
+
+import ensurepip
+
+test_tools.skip_if_missing('build')
+with test_tools.imports_under_tool('build'):
+    import bundle_ensurepip_wheels as bew
+
+
+# Disable fancy GitHub actions output during the tests
```

Comment:
There should be a test for enabled GHA, just for those helpers, then. It'd be sad not to get coverage there.